lib/holidays.rb

```diff
@@ -193,15 +193,14 @@ def year_holidays(options, from_date = Date.today)
       from_date = from_date.new_offset(0) + from_date.offset if from_date.respond_to?(:new_offset)
 
       from_date = get_date(from_date)
-      to_date = Date.new(from_date.year, 12, 31)
       regions, observed, informal = OptionFactory.parse_options.call(options)
 
       # This could be smarter but I don't have any evidence that just checking for
       # the next 12 months will cause us issues. If it does we can implement something
       # smarter here to check in smaller increments.
       date_driver_hash = UseCaseFactory.dates_driver_builder.call(from_date, from_date >> 12)
 
-      UseCaseFactory.year_holiday.call(from_date, to_date, date_driver_hash, regions, observed, informal)
+      UseCaseFactory.year_holiday.call(from_date, date_driver_hash, regions, observed, informal)
```

Comment:
I decided that the `to_date` could easily be calculated in the use case itself now that most of the logic was in the new 'search' logic and not in 'common context'.